### DISCUSSION TITLE
Code for IBM i in the 7.5 TR1 release notes

### DISCUSSION THREAD

**Discussion by worksofliam** [codefori/vscode-ibmi#900] [General]
Had to share because it's pretty cool to see!

![image](https://user-images.githubusercontent.com/3708366/195093896-44f22871-7409-435f-9262-f57b93594974.png)

https://www.ibm.com/common/ssi/ShowDoc.wss?docURL=/common/ssi/rep_ca/3/872/ENUSAP22-0273/index.html

**Comment by chrjorgensen** [comment]
@worksofliam Wow! Congratulations - your efforts are recognized! 🥇
